src/Console/InfectionContainer.php

```diff
@@ -71,7 +79,7 @@
 use PhpParser\PrettyPrinter\Standard;
 use Pimple\Container;
 use SebastianBergmann\Diff\Differ as BaseDiffer;
-use Symfony\Component\Console\Exception\InvalidArgumentException;
+use function sprintf;
```

Comment:
Maybe we could invest in a tool to replace all the native calls by the `Safe\*` ones?